tests/test_preprocessing/test_identitytransformer.py

```diff
@@ -24,5 +24,14 @@ def test_estimator_checks(test_fn):
 
 def test_same_values(random_xy_dataset_regr):
     X, y = random_xy_dataset_regr
-    X_new = IdentityTransformer().fit_transform(X)
+    X_new = IdentityTransformer(check_X=True).fit_transform(X)
     assert np.isclose(X, X_new).all()
+
+
+def test_nan_inf(random_xy_dataset_regr):
+    X, y = random_xy_dataset_regr
```

Comment:
It's a minor detail, and not a reason to block the PR ... 

... but in future PRs: it can be a good idea to have a docstring that links back to the GitHub issue. That way, if a test fails, there's a paper trail to the original issue that it's trying to fix.